### PR TITLE
#466 Integrate with Read the Docs

### PR DESCRIPTION
**Pull request by anilgursel**
Thanks for your pull request.  Please review the following guidelines.

- [x] Title includes issue id.
- [x] Description of the change added.
- [x] Commits are squashed.
- [ ] Tests added.
- [x] Documentation added/updated.
- [x] Also please review [CONTRIBUTING.md](https://github.com/paypal/squbs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,31 +24,6 @@ The easiest way to getting started is to create a project from one of the squbs
...
-* [Using the Orchestration DSL](docs/orchestration_dsl.md)
-* [The ActorRegistry](docs/registry.md)
-* [Timeout Policy](docs/timeoutpolicy.md)
-* [Validation](docs/validation.md)
-* [Persistent Buffer](docs/persistent-buffer.md)
-* [Console](docs/console.md)
-* [Application Lifecycle Management](docs/packaging.md)
-* [Streams Lifecycle](docs/streams-lifecycle.md)
-* [Pipeline](docs/pipeline.md)
-* [Resource Resolution](docs/resolver.md)
```
Comment (akara) [inline README.md]: Just a question. This index will be generated, right? I don't see it being moved anywhere else.

**Comment by akara** [comment]
Other than that, change looks good to me.

**Comment by anilgursel** [comment]
We already have all the links on the left panel.  So, not sure if we still need a TOC on the home page.  But, if you see the need, I can definitely add a TOC.

**Comment by akara** [comment]
I don't see the need, but I have not thoroughly reviewed the outcome, either. If you think we don't need one, I'm OK. Just want to make sure it is intentional.

File: docs/http-services.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Implementing HTTP(S) Services
 
+[TOC]
```
Comment (akara) [inline docs/http-services.md]: Hmm, this gets rendered into standard md as is. I'm not sure we can avoid that, though.

File: docs/http-services.md [hunk 3]
```diff
@@ -250,4 +252,16 @@ There are a few rules you have to keep in mind when implementing a `FlowDefiniti
...
\ No newline at end of file
+6. **Request path:** The `HttpRequest` object is handed to this flow unmodified. The `webContext` is in the `Path` of the request. It is the users job (as seen above) to handle the request with the knowledge of the `webContext`. In other words, the low-level API handles the `HttpRequest` directly and needs to manually take the web context into consideration for any path matching.
+
+## Metrics
+
+squbs comes with pre-built [pipeline](#pipeline) elements for metrics collection and squbs activator templates sets those as default.  Accordingly, each squbs http(s) service is enabled to collect [Codahale Metrics](http://metrics.dropwizard.io/3.1.0/getting-started/) out-of-the-box without any code change or configuration.  Please note, squbs metrics collection does NOT require AspectJ or any other runtime code weaving.  The following metrics are available on JMX by default:
```
Comment (akara) [inline docs/http-services.md]: 👍

File: docs/http-services.md (hunk 2 above)
Comment (anilgursel) [inline docs/http-services.md]: It is a feature of mkdocs.  Once rendered through mkdocs, it looks like below.  When I comment it out with `<!--- [TOC] --->`, then mkdocs ignores it.

screen shot 2017-06-02 at 12 59 52 pm

**Pull request merged**